### PR TITLE
Bring window to front once the scan is finished or stopped

### PR DESCRIPTION
**Pull request by mineminemine**
Adding this for #353. 
See `MainForm_Activate()`. Can be tested by doing a scan and either letting it finish or aborting it.

Also, I'm not sure why there wasn't proper indentation for the file; it was a mess previously, so it appears to be a lot of changes when there are only a few lines added.

### CODE REVIEW

File: InventoryKamera/ui/main/MainForm.cs [hunk 1]
```diff
@@ -17,529 +17,537 @@
...
+                    catch (ThreadAbortException)
+                    {
+                        // Workers can get stuck if the thread is aborted or an exception is raised
+                        if (!(data is null)) data.StopImageProcessorWorkers();
+                        UserInterface.SetProgramStatus("Scan stopped");
+                        MainForm_Activate();
+                    }
+                    catch (NotImplementedException ex)
+                    {
+                        UserInterface.AddError(ex.Message);
```
Comment (TitaniumLexa) [inline InventoryKamera/ui/main/MainForm.cs:254]: MainForm_Activate() is missed in this block if follow your approach

File: InventoryKamera/ui/main/MainForm.cs [hunk 2]
```diff
@@ -17,529 +17,537 @@
...
+                        MainForm_Activate();
+                    }
+                    finally
+                    {
+                        ResetUI();
+                        running = false;
+                        ManualExportButton.Invoke((MethodInvoker)delegate
+                        {
+                            ManualExportButton.Enabled = data.HasData;
+                        });
```
Comment (TitaniumLexa) [inline InventoryKamera/ui/main/MainForm.cs:272]: But i think it's better to call this function only in one place - finally block. Have you tried this?

**Comment by TitaniumLexa** [comment]
Oh, and I think it'll be better if it was split into 2 separate commits for readability - indentation fix and feature.

File: InventoryKamera/ui/main/MainForm.cs (hunk 2 above)
Comment (mineminemine) [reply InventoryKamera/ui/main/MainForm.cs:269]: Right didn't of that, I readjusted it to only call for activate in `finally`

**Comment by mineminemine** [comment]
will reopen a new PR with cleaner commits

**Pull request closed without merging**